src/pages/data-globe/data-globe.js

```diff
@@ -5,9 +5,10 @@ import { layerManagerToggle, exclusiveLayersToggle, layerManagerVisibility, laye
 import Component from './data-globe-component.jsx';
 import mapStateToProps from './data-globe-selectors';
 import biodiversityActions from 'redux_modules/biodiversity-data/biodiversity-data';
+import pageTextsActions from 'redux_modules/page-texts/page-texts';
```

Comment:
I think this is not needed here (as we are importing the action on the `connect`ed components `minimap-widget` and `about`)